option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message HeroCultureReplaceRequest
{
 required U64Type guid = 1;
}
